tmt/cli.py

```diff
@@ -687,6 +687,28 @@ def create(context, name, template, force, **kwargs):
     tmt.Plan.create(name, template, context.obj.tree.root, force)
 
 
+@plans.command()
+@click.pass_context
+@name_filter_condition_long
+@click.option(
+    '--format', 'format_', default='yaml', show_default=True, metavar='FORMAT',
```

Comment:
I'd suggest specifying the choices in the CLI as well, see https://click.palletsprojects.com/en/8.0.x/options/#choice-options

Reply:
Hm, that's maybe also relevant to the other export methods now that I'm looking into it :)

Reply:
Yes, agreed to address this in a separate pull request for all `export` methods.

Reply:
Let's track this in #1084.